src/Valitron/Validator.php

```diff
@@ -857,6 +862,19 @@ protected function getPart($data, $identifiers)
     }
 
     /**
+     * Force Validate Empty Values
+     *
+     * @param boolean $flag
+     */
+    public function forceValidateEmptyValues($flag)
+    {
+        if (!is_bool($flag)) {
+            throw new LogicException('invalid parametor has specified');
```

Comment:
Throw a new `\InvalidArgumentException` here with the message: 'First parameter must be a boolean true/false'.